src/Compute/Compute.Test/SessionRecords/Microsoft.Azure.Commands.Compute.Test.ScenarioTests.VirtualMachineScaleSetTests/TestVirtualMachineScaleSetReimageUpdate.json

```diff
@@ -4,21 +4,21 @@
       "RequestUri": "/subscriptions/e37510d7-33b6-4676-886f-ee75bcc01871/providers/Microsoft.Compute?api-version=2016-09-01",
```

Comment:
Is your changes affected this test case? If not you can revert the change and run the CI pipeline again

Reply:
No it does not I will revert and try again.